games/s.yaml

```
@@ -1,3 +1,27 @@
+- name: SanAndreasUnity
+  type: remake
+  originals:
+  - "Grand Theft Auto: San Andreas"
+  repo: https://github.com/GTA-ASM/SanAndreasUnity
+  url: https://github.com/GTA-ASM/SanAndreasUnity
+  development: sporadic
+  lang: C#
+  status: playable
+  multiplayer:
+  - Online
+  - LAN
+  framework: Unity
+  license:
+  - MIT
+  content: open
```

```suggestion
  content: commercial
```
it requires original game data